tests/functional/testOnboarding.js

```diff
@@ -68,13 +59,17 @@ describe('Onboarding', function() {
      });
 
   it('Sign up button opens auth flow', async () => {
+    // must wait for animation
+    await vpn.wait()
```

Comment:
See my next comment

Reply:
I will create a fast follow to investigate this issue with interaction here.